### ISSUE TITLE
HCM: Missing hover color in High Contrast Mode for `post-accordion` component

### ISSUE THREAD

**Issue by alionazherdetska** [swisspost/design-system#4283]
**Description:**
The hover color for interactive elements in high contrast mode is not applied for

1. .accordion-header
2. post-icon name='2051'

**Steps to Reproduce:**

- Enable high contrast mode in the browser or system settings.
- Navigate to the `post-accordion` component page
- Hover over the elements.
